tests/atomistics/structure/test_atoms.py

```diff
@@ -1370,6 +1371,24 @@ def test_non_periodic(self):
             c3.get_scaled_positions()
             self.assertEqual(len(w), 0)
 
+    def test_pyiron_ase_conversion(self):
+        ase_struct = pyiron_to_ase(self.CO2)
+        self.assertIsInstance(ase_struct, ASEAtoms)
+        self.assertNotIsInstance(ase_struct, Atoms)
+        pyiron_struct = ase_to_pyiron(ase_struct)
+        self.assertNotIsInstance(pyiron_struct, ASEAtoms)
+        self.assertIsInstance(pyiron_struct, Atoms)
+
+    def test_write(self):
+        struct = create_ase_bulk("Al", cubic=True)
+        self.assertIsInstance(struct, Atoms)
+        self.assertNotIsInstance(struct, ASEAtoms)
+        struct.write("POSCAR_test")
+        os.remove("POSCAR_test")
+        struct = create_ase_bulk("Al")
+        struct.write("POSCAR_test")
+        os.remove("POSCAR_test")
```

Comment:
It seems like the tests are trustworthy now, right? Or are they still sensitive to the order of imports?

Reply:
They are. Now the order of imports don't matter!